doc/source/user_guide/label_guide.rst

```diff
@@ -353,12 +360,13 @@ keys are dimension names and values coordinate labels and pass that to the paren
     @savefig custom_plot_posterior1.png
     az.plot_posterior(idata, coords=coords, labeller=labeller);
 
-This has the advantage of requiring very little extra code and to allow to combine
-our newly created ``NonIdxCoordLabeller`` with other labellers like we did in
-the previous section.
+This has the following advantages:
+
+- It requires very little extra code.
+- It allows to combine our newly created ``NonIdxCoordLabeller`` with other labellers as we did in the previous section.
 
 Another option is to go for a much more customized look, and handle everything
-on ``make_label_vert`` to get labels like "Correlation between subjects x and y".
+on :class:`~arviz.labels.IdxLabeller.make_label_vert` to get labels like "Correlation between subjects x and y".
```

Comment:
if changing these to a cross-reference, they should refer to `BaseLabeller` methods (same below), not to IdxLabeller. 

Side note, it also seems to work like that, but `make_label_vert` and `make_label_flat` are methods, not classes, so their ref type should be `meth`